list/lib/listitem/list-item-only.ts

```diff
@@ -26,4 +26,8 @@ export class ListItemOnly extends ListItemEl {
   override renderRipple() {
     return this.noninteractive ? nothing : super.renderRipple();
   }
+
+  override renderFocusRing() {
```

Comment:
Uh oh, build errors! We need to add an explicit type to the `super.renderFocusRing()`, just like [`renderRipple()` does](https://github.com/material-components/material-web/blob/33daf19c73ae1599a10286dada80edf288b52011/list/lib/listitem/list-item.ts#L144).

Update `list-item.ts` and change that method return type to

```
protected renderFocusRing(): TemplateResult|typeof nothing {
```

Reply:
Aha, Think it's best I just open it up in vs code instead of editing on GH 😅

Reply:
@VandeurenGlenn it's just one line :eyes:

Reply:
Yes well, @vdegenne the rest wasn't also much but still gave an error :p

Reply:
@VandeurenGlenn well you messed up :rofl:

Reply:
iono what happened, but our internal copy tool did NOT like that 😂

Reply:
@asyncLiz Done, also made noninteractive not clickable